x/marker/types/msgs.go

```diff
@@ -33,12 +33,18 @@ var AllRequestMsgs = []sdk.Msg{
 	(*MsgSetDenomMetadataRequest)(nil),
 	(*MsgGrantAllowanceRequest)(nil),
 	(*MsgAddFinalizeActivateMarkerRequest)(nil),
-	(*MsgSupplyIncreaseProposalRequest)(nil),
 	(*MsgUpdateRequiredAttributesRequest)(nil),
 	(*MsgUpdateForcedTransferRequest)(nil),
 	(*MsgSetAccountDataRequest)(nil),
 	(*MsgUpdateSendDenyListRequest)(nil),
 	(*MsgAddNetAssetValuesRequest)(nil),
+
+	(*MsgSupplyIncreaseProposalRequest)(nil),
+	(*MsgSupplyDecreaseProposalRequest)(nil),
+	(*MsgChangeStatusProposalRequest)(nil),
+	(*MsgSetDenomMetadataProposalRequest)(nil),
+	(*MsgSetAdministratorProposalRequest)(nil),
+	(*MsgRemoveAdministratorProposalRequest)(nil),
```

Comment:
Missing entry for `WithdrawEscrowProposal`.

Also, please order these the same way they are in the `.proto` file.

Reply:
Good catch, thanks!